common/script/fns/predictableRandomNamespaced.js

```diff
@@ -0,0 +1,5 @@
+import predictableRandom from './predictableRandom';
```

Comment:
We can't export an object of functions because there are fns that require other fns, libs that require other libs, ... so they would cause circular dependencies